### ISSUE TITLE
FAQ questions for Wiki

### ISSUE THREAD

**Issue by kokosek** [Code-Fights/codefights-wiki#35]
* In tournaments, does each pair in a round get the same puzzles?
* Are challenges added on specific times of a day? If so, when?
* What does "first" in a challenge mean? Being first in the ranking or first to solve the challenge? Either way, does all languages count as one or can there be 4 first users?
* What's the idea behind CodeFights logo?

More to be added.

**Comment by mikesobol** [comment]
- Can I solve a challenge using two or more languages?

**Comment by lonerz** [comment]
Yes @mikesobol. Your most recent submission is the one that counts for the prize money though.

**Comment by lonerz** [comment]
@kokosek Here are most of the answers:

- In tournaments, everyone gets the same questions.
- Two challenges are usually added everyday. I think (1:00 PM PDT and 1:00 AM PDT), but not sure.
- "First" in a challenge means the first person out of **all** languages to solve the challenge. This does not include the problem creator.
- CodeFights logo was based on the MathFights logo: mathfights.com
- Yes, there can be more than 1 tournament at a time. You can create your own private tourneys now.
- Yes, you can create tournaments now.
- Contributor badges are only for admins currently.
- 1000 coins: one day, 2000 coins: one day, 3000: two days, 4000: 2-3 days. These are average times.
- Solutions in challenges that are greyed out mean that they either were blocked by an admin, the person submitted a solution in another language, the person was the challenge-creator, or an admin that was testing the challenge.
- Coins: for a regular fight (random): you get the amount of coins equal to your score (i think). For a friendly fight, it is half of that. Challenges: self-explanatory. Tournaments (created by CodeMaster): 2000 (1st), 1000 (2nd), 500 (3rd), 1/2^(n-1)*2000 where n is your place.
- The colors near the CodeFriends indicate the status of your friend. Green- online and not playing a fight. Red - online, but in a fight currently. Grey- offline.
- You can change your default language by looking at your profile on the left side and clicking your language.
- You can delete your account by emailing info@codefights.com
Hope this helps! Totally welcome to answer other questions.

**Comment by kokosek** [comment]
Hello. Thanks for all the answers. :-)
But I'm not closing the issue because I'd still add a FAQ section to CodeFights or at least Wiki.
